Makefile

```diff
@@ -33,6 +33,9 @@ start: ## Start the containers
 stop: ## Stop the containers
 	${DOCKER_COMPOSE} down
 
+initdb: ## Setup database tables
+	${DOCKER_COMPOSE} run app npm run initdb
```

Comment:
`run --rm`?

Reply:
It's good for this to be available as an `npm` target in the image; this will probably be an `initContainer` in Kubernetes that is run before the Node.js application is started